com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/topology.py

```diff
@@ -72,6 +72,21 @@ def transform(self, func):
         op.addInputPort(outputPort=self.oport)
         oport = op.addOutputPort()
         return Stream(self.topology, oport)
+     
+    def multiTransform(self, func):
+        """
+        Transforms each tuple from this stream into 0 or more tuples using the supplied function.
+        For each tuple on this stream, the returned stream will contain all non-None tuples in
+        the iterator that is the result of the supplied function.
+        Tuples will be added to the returned stream in the order the iterator
+        returns them.
+        If the return is None or an empty iterator then no tuples are added to
```

Comment:
Added the following comment for multiTransform():
The supplied function must return an iterable, otherwise a TypeError is raised.